build/images/deps/cni-binaries-version

```diff
@@ -1 +1 @@
-v0.8.7
+v1.1.1
```

Comment:
why not use the latest version (v1.1.2)?

Reply:
will bump, thanks

Reply:
the latest version is 1.1.1, https://github.com/containernetworking/plugins/releases @antoninbas

Reply:
my bad, I must have been looking at the wrong repo